test/inductor/test_torchinductor.py

```diff
@@ -146,6 +146,18 @@ def int(self):
         return torch.arange(self.n, device=self.device, dtype=torch.int32)
 
 
+def cpp_wrapper(func):
+    @functools.wraps(func)
+    def wrapTheFunction(*args):
+        for cpp_wrapper_enabled in [True, False]:
+            old = config.cpp_wrapper
+            config.cpp_wrapper = cpp_wrapper_enabled
+            func(*args)
+            config.cpp_wrapper = old
+
+    return wrapTheFunction
```

Comment:
@patch.object(config, "cpp_wrapper", True)?

Reply:
Refactored the UT to use `patch.object`